### DISCUSSION TITLE
Looking for an active maintainer

### DISCUSSION THREAD

**Discussion by janbaykara** [janbaykara/ebook-scraper#12] [General]
Hey all. This repo has received a small handful of issues and emails which is exciting!

I developed this tool when I was a student frantically reading everything I could get my hands on. I don't have institutional access to ebook readers anymore or much of a motivation to keep on top of this repo, but if someone else has found this tool useful and would like to step up as a regular contributor, I'd be happy to add you as an admin to this repo.

Can you volunteer to help maintain this repo?

**Comment by obfuscatedgenerated** [comment]


**Reply by janbaykara** [reply]
Hey, that's really cool to hear. I'd be happy to hand off so that I'm not blocking development, especially if you have access to a book site and an understanding of the basic need of the tool.

I've invited you to collaborate on the repo.

If you're up for it, we could arrange a quick sync up call later down the line.

**Comment by danitt** [comment]
hi @janbaykara I've found this repo quite helpful and I feel confident I could carry the torch forward with it

I notice there's a decent looking [PR](https://github.com/janbaykara/ebook-scraper/pull/55) submitted recently which would be a great start

happy to hop on a call if you like - lmk!

**Reply by janbaykara** [reply]
Hi both -- really glad to hear from you both and apologies for the silence. Incredibly busy and haven't noticed all this.

I would love it if someone wants to pick this up. Wonder if people fancy a quick call? Maybe you could both email me at janbaykara@pm.me, I can make time from Wednesday on next week.

Alternatively you can use my work booking link: https://calendar.app.google/ckTZhsHzt2zcgmB58

**Reply by janbaykara** [reply]
Update on this that @danitt and I had a good call and we're going to be working together to get this extension warmed back up, and hopefully submitted to the Chrome Web Store in due course! Massive props to Dan :)

**Comment by cornercutgaming-gif** [comment]
Hi,
This is great - however, I have been getting the error "invalid string length" when trying to download a book. Note that the book is over 1,800 pages. Any suggestions on why this is happening / on fixing it?
